relay-dynamic-config/src/defaults.rs

```diff
@@ -395,5 +395,73 @@ fn span_metrics(is_extract_all: bool) -> impl IntoIterator<Item = MetricSpec> {
                     .always(), // already guarded by condition on metric
             ],
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "c:spans/dynamically_loaded_libraries_count@none".into(),
```

Comment:
We call the thing we're counting debug images or [DIFs](https://docs.sentry.io/product/cli/dif/) internally, so I would stick with that name, i.e. `c:spans/dif_count@none` or `c:spans/debug_image_count@none`.

Also make sure to register the name of the new metric in [strings.py](https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/indexer/strings.py) before merging this PR.